data/moves.ts

```diff
@@ -13230,11 +13230,8 @@ export const Moves: {[moveid: string]: MoveData} = {
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1},
-		onUseMoveMessage(source, target, move) {
```

Comment:
It might be possible to just use a different event here; `TryHit` might work (the `singleEvent` version runs shortly before a move hits) or there's always `AfterDamage` and `AfterSubDamage`.